java/maven/src/org/netbeans/modules/maven/runjar/RunJarPrereqChecker.java

```diff
@@ -99,6 +105,58 @@
         return true;
     }
 
+    private static String findMainClass(RunConfig config) {
+        // Read main class from the manifest property:
+        String mainClass = getConfiguration(config, "maven-jar-plugin", "archive", "manifest", "mainClass");
```

Comment:
maybe add `// NOI18N` as these texts shall never be localized.

Reply:
Added.